Apps/Admin/Server/Controllers/TagController.cs

```diff
@@ -94,17 +95,17 @@ public RequestResult<AdminTagView> DeleteTag([FromBody] AdminTagView tag)
         /// <summary>
         /// Associate an existing admin tag to the feedback with matching id.
         /// </summary>
-        /// <returns>The added tag model wrapped in a request result.</returns>
+        /// <returns>Returns the associated admin tag wrapped in a request result.</returns>
```

Comment:
Nit: the type of the return value is not accurately reflected in the documentation.

Reply:
Changed.